docs/conf.py

```diff
@@ -62,3 +62,5 @@
 html_theme_options = {
     "navigation_depth": 4,
 }
+
+epsilon: float = 1e-12
```

Comment:
defining epsilon somewhere once & re-using it is a good idea, but this file is not the place to do it (not part of the python package & only used for generating the docs, so importing this file will fail for installed vstt) - probably defining `epsilon` in src/vstt/stats.py would make sense - maybe with a slightly more descriptive name since it is now used in a wider scope?